static/js/chart/draw_d3_map.ts

```diff
@@ -511,7 +489,12 @@ export function addMapPoints(
     });
   const minDotSize = minDotRadius || Math.max(minRegionDiagonal * 0.02, 1.1);
   const filteredMapPoints = mapPoints.filter((point) => {
-    return !_.isNull(projection([point.longitude, point.latitude]));
+    const projectedPoint = projection([point.longitude, point.latitude]);
+    return (
+      projectedPoint &&
+      !_.isNaN(projectedPoint[0]) &&
+      !_.isNaN(projectedPoint[1])
```

Comment:
did you dig into which data points were returning NaN here? there might be some underlying data issues we should fix.

Reply:
dug into this and looked like it was because mixer was returning undefined for 0 values. have a pr up to fix this: https://github.com/datacommonsorg/mixer/pull/1024